src/ec/packages/ec-component-site-header/site-header.html.twig

```diff
@@ -0,0 +1,129 @@
+  },
+  button: {
+    variant: 'search',
+    icon: {
+      type: 'general',
+      name: 'search',
+      path: _icon_file_path,
+    },
+    label: 'Search',
+    extra_classes: 'ecl-search-form__button'
+  }
+}
+%}
+
+{% set _language = {
+  url: '',
+  label: '',
+  code: ''
+}
+%}
+
+{% set _header_link = {
+  url: '',
+  aria_label: ''
+}
+%}
+
+{% set _header_image = {
+  src: '',
+  alt: '',
+  title: '',
+}
+%}
+
+{% set _css_class = 'ecl-site-header' %}
+{% set _extra_attributes = '' %}
+{% set _icon_file_path = icon_file_path|default('') %}
+
+{# Merge options #}
+{% if language is defined and language is not empty %}
+  {% set _language = _language|merge(language) %}
+{% endif %}
+
+{% if header_link is defined and header_link is not empty %}
+  {% set _header_link = _header_link|merge(header_link) %}
+{% endif %}
+
+{% if header_image is defined and header_image is not empty %}
+  {% set _header_image = _header_image|merge(header_image) %}
+{% endif %}
+
+{% if search_form.text_input is defined and search_form.text_input is not empty %}
+  {% set _search_form = _search_form|merge({ text_input: _search_form.text_input|merge(search_form.text_input) }) %}
+{% endif %}
+
+{% if search_form.button is defined and search_form.button is not empty %}
+  {% set _search_form = _search_form|merge({ button: _search_form.button|merge(search_form.button) }) %}
+{% endif %}
+
+{# Internal logic - Process properties #}
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+{% spaceless %}
+<header class="{{- _css_class -}}"{{- _extra_attributes|raw -}}>
+  <div class="ecl-site-header__container ecl-container">
+    <div class="ecl-site-header__banner">
+      <a class="ecl-link ecl-link--standalone" href="{{- _header_link.url -}}" aria-label="{{- _header_link.aria_label -}}">
+        <img alt="{{- _header_image.alt -}}" title="{{- _header_image.title -}}" class="ecl-site-header__logo-image" src="{{- _header_image.src -}}" />
+      </a>
+      <div class="ecl-site-header__selector">
+        <a class="ecl-link ecl-link--standalone" href="{{- _language.url -}}">
+          {{- _language.label -}}
+          <span class="ecl-site-header__language-icon">
+            {%- include '../ec-component-icon/icon.html.twig' with { icon: { path: _icon_file_path, type: 'general', name: 'language' } } -%}
+            <span class="ecl-site-header__language-code">{{- _language.code -}}</span>
+          </span>
+        </a>
+      </div>
+    </div>
+    {%- include '../ec-component-search-form/search-form.html.twig' with _search_form -%}
```

Comment:
missing class `ecl-site-header__search` here

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/33/commits/659a80d94c84289694dbd0bf95463b6209c291d4

I had to modify the file `search-form.html.twig` in `EC Search Form` in https://github.com/ec-europa/ecl-twig/pull/33/commits/c89d15dd37e60844a2b59de248661b6b266c1378 because I founded a small bug there. But it does not affect the tests or other files so I think we can approve it here.